config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")  
-  #root "users#index"
+  root to: "users#index"
   
   get "/users/:id/posts", to: "posts#index"
   get "/users/:user_id/posts/:id", to: "posts#show"
```

Comment:
- [ ] [optional] @Team it would have been nice if you used resources for your routes, that way you could make your routes code short and neat